src/current/v23.2/backup-and-restore-monitoring.md

```diff
@@ -7,26 +7,19 @@ docs_area: manage
 
```

Comment:
This is the self-hosted backup monitoring page. Here the work was removing Cloud content.